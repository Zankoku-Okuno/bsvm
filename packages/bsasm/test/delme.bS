;comment

  .shebang /usr/bin/env bsvm
  @foo:
   hcf