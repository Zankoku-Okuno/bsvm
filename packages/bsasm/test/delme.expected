;comment

  .shebang /usr/bin/env bsvm
t01 `shebang`
  @foo:
t00 `@foo`
   hcf
t02 `hcf`
